app/main.py

```diff
@@ -1,3 +1,25 @@
+import math
+
+
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        full_weeks = 0
```

Comment:
Here you can use only this:
return math.ceil(days / 7)
Of course, you have to import math before